pytensor/tensor/nlinalg.py

```diff
@@ -228,7 +215,7 @@ def __str__(self):
         return "Det"
 
 
-det = Det()
+det = Blockwise(Det())
```

Comment:
is there a better way to always have blockwise ops?

Reply:
This is the same as `Elemwise` always creating from `Scalar` when you import PyTensor